src/main/java/uk/gov/digital/ho/hocs/application/properties/AwsSqsProperties.java

```diff
@@ -0,0 +1,53 @@
+package uk.gov.digital.ho.hocs.application.properties;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+import uk.gov.digital.ho.hocs.application.properties.model.Account;
+import uk.gov.digital.ho.hocs.application.properties.model.Config;
+
+import javax.validation.constraints.Max;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+
+@Configuration("awsSqsConfig")
+@ConfigurationProperties(prefix = "aws.sqs")
+@Getter
+@Setter
+public class AwsSqsProperties {
+
+    private Config config;
+
+    private Queue ukviComplaint;
+
+    @Getter
+    @Setter
+    public static class Queue {
+
+        @NotBlank
+        private String url;
+
+        private Account account;
+
+        private Attributes attributes;
+
+        @Getter
+        @Setter
+        public static class Attributes {
+
+            @Min(1)
+            @Max(10)
```

Comment:
not a blocking commit, but I don't feel these are needed, just get the config right - its not a user input.

Reply:
It comes down to how explicit you want to be, hovering over this properties in the app props tells you it's expected values. Last thing we want it someone putting 20 in thats not accepted and spending time investigating something we can be explicit about now.

Reply:
Hmm I decided to remove the config properties (but keep these attributes within the app properties). Max messages would be useful if we need to control the flow in production for some reason.